src/biometryType.android.js

```diff
@@ -0,0 +1,12 @@
+import { NativeModules } from 'react-native';
+import createError from './createError';
+
+const { ReactNativeFingerprintScanner } = NativeModules;
+
+export default () => {
+  return new Promise((resolve, reject) => {
+    ReactNativeFingerprintScanner.biometryType()
```

Comment:
And then we can just do `return ReactNativeFingerprintScanner.biometryType()` here.